spinn_machine/machine.py

```diff
@@ -645,3 +645,21 @@ def total_cores(self):
         """
         return len([
             processor for chip in self.chips for processor in chip.processors])
+
+    def has_wrap_arounds(self):
```

Comment:
None really - just affects whether you call it as a property or not.  As it has no arguments and looks like a property, probably worth making it one.  It has no side-effects either, so it does make sense.